CHANGELOG.md

```diff
@@ -7,7 +7,39 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - [#17] Add elapsed attribute to ReturnEvent
-- [#8] add git to metadata
+- [#8] Add git to metadata
+- pytest integration
+- Support `APPMAP_OUTPUT_DIR` environment variable. If set, specifies the root directory
+  for writing AppMaps. If not set, defaults to `tmp/appmap`.
+
+### Changed
+- Use repr() instead of str() for receiver
```

Comment:
@apotterri Changelog should be a summary of changes since last release. I believe these 2 entries could be skipped

Reply:
Hmmm. I think all the entries here are still unreleased. Which two entries did you mean?

Reply:
@apotterri sorry, was confused. Please ignore.